model/Index/Service/SyncResultIndexer.php

```diff
@@ -49,7 +49,7 @@ public function addIndex($resource): void
         $normalizedResource = $this->normalizer->normalize($resource);
 
         try {
-            $document = $this->getIndexerService()->getDocumentBuilder()->createDocumentFromArray(
+            $document = $this->getLegacyIndexDocumentBuilder()->createDocumentFromArray(
```

Comment:
Maybe we should call `AdvancedSearchIndexDocumentBuilder` instead? Wdyt?

Reply:
fixed

Reply:
Thanks!